src/plugins/credential_management/server/saved_objects/credential_saved_objects_type.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { SavedObjectsType } from 'opensearch-dashboards/server';
+
+export const credentialSavedObjectType: SavedObjectsType = {
```

Comment:
Will migrations be required?

Reply:
watching this comment :)

Reply:
> Will migrations be required?

There are no previous saved objects to migrate because it is a new typed saved object. However, a new issue was created to track this as needed. https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2046